pkg/cmd/admin.go

```diff
@@ -218,5 +229,15 @@ func NewRestoreCommand() *cobra.Command {
 		},
 	}
 
+	cmd.Flags().BoolVar(&autorec, "auto-recovery", false,
+		`Changes the behavior of the restore command.
+The PATH argument will be treated as a directory where backups are
+created with "backup --auto-recovery" command.`)
+
+	cmd.Flags().BoolVar(&saveFailed, "save-failed", true,
+		`Only applicable if --auto-recovery is also specified.
+It true (default), a copy of MicroShift data directory will be made inside
```

Comment:
```suggestion
If true (default), a copy of MicroShift data directory will be made inside
```
is this a typo, right?

Reply:
Yes, thx